README.md

```diff
@@ -21,27 +21,19 @@
 
 *Support for Table accounts in CosmosDB is coming soon!
 
-## Mongo "Scrapbooks"
+![Browse CosmosDB and MongoDB databases](resources/Browse.png)
 
-- Configure the user setting `mongo.shell.path` to your mongo shell executable path
-- Click on any DB to open the Mongo shell playground editor
-- Enter your scripts, eg: `db.<collectionName>.find()`
-- IntelliSense (completions) will be provided as you write your scripts
-- Select the script and press `CMD+"` (`CTRL+"` on Windows and Linux) to see the results
+## Mongo "Scrapbooks" (Run Mongo Commands with Rich Intellisense)
 
-## Managing Azure Subscriptions
-
-If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
-
-![Sign in to Azure](resources/SignIn.gif)
-
-If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "Create a Free Azure Account..." or selecting "View->Command Palette" and searching for "Azure: Create an Account".
+- Attach to a MongoDB database account
+- Optionally configure the user setting `mongo.shell.path` to your mongo shell executable path (many of the common commands have built-in support and do not require the Mongo shell to be installed - see [Prerequisites](#prerequisites))
+- Right-click on a database and select "Connect"
+- Click on "New Mongo Scrapbook" in the tree title bar
+- Enter your script, eg: `db.<collectionName>.find()`

```

Comment:
We should change all instances of 'script' to 'command'

Script implies multiple commands, but we only support one command at a time. "Command" is also consistent with our naming inside the extension:
![screen shot 2018-03-29 at 9 06 34 am](https://user-images.githubusercontent.com/11282622/38099855-81445218-3330-11e8-931f-fd93a5d4ccce.png)